build-support/travis/travis.yml.mustache

```diff
@@ -293,6 +334,21 @@ py36_linux_build_engine: &py36_linux_build_engine
     - CACHE_NAME=linuxpexbuild.py36
     - BOOTSTRAPPED_PEX_KEY_SUFFIX=py36.linux
 
+# NB: we do not use Docker for Py37 because Centos6 has too outdated of OpenSSL. Since we only use Py37 for
```

Comment:
Can you please make this a TODO and link to an issue describing the OpenSSL incompatibility?

Reply:
I would possibly just move the text you have in this PR description describing that into its own issue, and link to that in the description and in this comment.

Reply:
Filed https://github.com/pantsbuild/pants/issues/7421 and will update this comment next time I push.